package-lock.json

```diff
@@ -0,0 +1,3860 @@
+{
```

Comment:
do you mind removing this file? I just happen to be using yarn in this project instead

Reply:
Sure, I'm not too familiar with npm and CI, but included the file because the command line told me to.